ray-operator/controllers/ray/rayservice_controller.go

```diff
@@ -1104,33 +1099,6 @@ func (r *RayServiceReconciler) reconcileServices(ctx context.Context, rayService
 	return nil, err
 }
 
-func (r *RayServiceReconciler) updateStatusForActiveCluster(ctx context.Context, rayServiceInstance *rayv1.RayService, rayClusterInstance *rayv1.RayCluster) error {
-	logger := ctrl.LoggerFrom(ctx)
-	rayServiceInstance.Status.ActiveServiceStatus.RayClusterStatus = rayClusterInstance.Status
```

Comment:
This will immediately be overwritten with the pending cluster status by the subsequent `reconcileServe`. In addition, in #2826, `RayClusterStatus` will be set in `calculateStatus` instead.